src/document/mod.rs

```diff
@@ -1,3 +1,7 @@
+//! Document types
```

Comment:
Be aware that this gets combined with the comment block in `lib.rs` on `pub mod document`. I think I slightly prefer to put the docs here. We could just make the ones in `lib.rs` `//` comments, or probably just remove them.

Reply:
![image](https://user-images.githubusercontent.com/19213143/82588698-690b5880-9b58-11ea-8048-bbb6d3f18a39.png)

Reply:
Sorry, this is a result of my bad "git-ness". I had some changes in the group PR and some in the document PR, and did a bad job of separating them. I believe the group PR removes all of the `lib.rs` `///` comments and switches to individual file comments. 
I'll be sure it all merges together correctly.